docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -956,6 +956,11 @@ instance_limits:
 # limiting feature.)
 # CLI flag: -distributor.reusable-ingester-push-workers
 [reusable_ingester_push_workers: <int> | default = 2000]
+
+# (advanced) Change the implementation of Otel startTime from a real zero to a
```

Comment:
```suggestion
# (advanced) Change the implementation of OTel startTime from a real zero to a
```

Reply:
Thanks.  This flag is not supposed to appear in the docs; I will need to research how to achieve that.